### PR TITLE
HPCC-12952 WU/DFUWU Refresh on Idle

### PR DESCRIPTION
**Pull request by GordonSmith**
Refresh WU lists on an idle timer.

Fixes HPCC-12952

Signed-off-by: Gordon Smith gordon.smith@lexisnexis.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

File: esp/src/eclwatch/GetDFUWorkunitsWidget.js [hunk 1]
```diff
@@ -84,6 +84,20 @@ define([
 
         startup: function (args) {
             this.inherited(arguments);
+            this.initContextMenu();
+            this._idleWatcher = new ESPUtil.IdleWatcher();
+            this._idleWatcher.start();
+            var context = this;
+            this._idleWatcherHandle = this._idleWatcher.on("idle", function () {
+                console.log("idle");
```
Comment (miguelvazq) [inline esp/src/eclwatch/GetDFUWorkunitsWidget.js]: remove console message

File: esp/src/eclwatch/WUQueryWidget.js [hunk 2]
```diff
@@ -84,6 +84,19 @@ define([
         startup: function (args) {
             this.inherited(arguments);
             this.initContextMenu();
+            this._idleWatcher = new ESPUtil.IdleWatcher();
+            this._idleWatcher.start();
+            var context = this;
+            this._idleWatcherHandle = this._idleWatcher.on("idle", function () {
+                console.log("idle");
```
Comment (miguelvazq) [inline esp/src/eclwatch/WUQueryWidget.js]: remove console message

**Comment by miguelvazq** [comment]
Just the two comments mentioned above. All else looks good.

**Comment by GordonSmith** [comment]
Updated as per comments.

**Pull request merged**